bot.js

```diff
@@ -1,35 +1,59 @@
 var Discord = require('discord.io');
 var logger = require('winston');
-var auth = require('./auth.json');
+var  auth = require('./auth.json');
 
-// Configure logger settings
-logger.remove(logger.transports.Console);
-logger.add(logger.transports.Console, {
+//Configure logger settings
+logger.remove(logger.transports.Console, {
     colorize: true
 });
-
 logger.level = 'debug';
 
-// Initialize Discord Bot
+//Канал, в котором бот может реагировать на сообщения
```

Comment:
```suggestion
// Канал, в котором бот может реагировать на сообщения
```
Put a space after `//`.